### ISSUE TITLE
Make the client- and serverside builds the same

### ISSUE THREAD

**Issue by marcveens** [macaw-interactive/react-thingy#3]
Currently we compile the clientside bundle to ES5 and the serverside bundle to ES2018. It might be a good idea to keep those equal to each other. This way we have a certainty that the code behaves the same everywhere. 

An issue I ran across was the creating an instance of a class with properties in it, like this for example:
```javascript
public class Car {
 public wheels: number;
 public doors: number;
}
```
Clientside we can create a new instance of `Car` and access the `wheels` and `doors` properties.   

Serverside we can create a new instance of `Car`, but the instance doesn't contain the properties anymore.

**Comment by svdoever** [comment]
That would be a pity, because at server-side we have native implementations like async/await which would be transpiled to 15 years old ES5 compatible JavaScript...

**Comment by marcveens** [comment]
True, but like I said, that's the only way we can guarantee both versions behaving the same. Or we can choose to drop IE11 and use async/await in browsers as well..

**Comment by svdoever** [comment]
Not a good idea:-) maybe introduce tests to guarantee code behaves te same?

Outlook voor Android downloaden<https://aka.ms/ghei36>

________________________________

**Comment by marcveens** [comment]
But we're not going to test if we can create an instance of a class, right? There might be some other weird behaviors which we don't know about just yet. 

Performance-wise ES6 is not necessarily faster than ES5 based on http://incaseofstairs.com/six-speed/. I took a look at the V8 engine and ES6 implementation to make a comparison. So that shouldn't be a big problem. Yes, it's an older technique, but I don't think we'll notice the difference since we're still developing using ES6.
